frontend/src/FilterLists/ExportFilterList.tsx

```diff
@@ -0,0 +1,42 @@
+import { Intent, MenuItem } from '@blueprintjs/core';
+import { useState } from 'react';
+
+import { ExportFilterList as ExportFilterListBackend } from '../../wailsjs/go/files/Files';
+import { AppToaster } from '../common/toaster';
+
+
+export function ExportFilterList() {
+  const [loading, setLoading] = useState(false);
+
+  const handleExport = async() => {
+    setLoading(true);
+    try {
+      const result = await ExportFilterListBackend();
+
+      const blob = new Blob([JSON.stringify(result)], { type: 'application/json' });
+      const url = URL.createObjectURL(blob);
+      const link = document.createElement('a');
+      link.href =  url;
+      link.download = 'filter-lists.json';
+      document.body.appendChild(link);
+      link.click();
+
+      document.body.removeChild(link);
+      URL.revokeObjectURL(url);
+    } catch (error) {
+      AppToaster.show({
+        message: error instanceof Error ? error.message : 'Failed to export filter lists',
+        intent: Intent.DANGER,
+      });
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return <MenuItem 
+    icon="download"
```

Comment:
Swapping icons (`upload` for `ExportFilterList` menu item, `download` for `ImportFilterList` menu item) makes the their intent more clear, IMO.